app/models/katello/content_override.rb

```diff
@@ -0,0 +1,36 @@
+module Katello
+  class ContentOverride
+    attr_accessor :content_label, :name, :value
+    ENABLEMENT_VALUES = ["yes", "no", "0", "1", "default"].freeze
+
+    def initialize(content_label)
+      @content_label = content_label
+    end
+
+    def enabled=(value = nil)
+      @name = "enabled"
+      @value = value
+    end
+
+    def to_hash
+      ret = {"content_label" => @content_label, "name" => @name}
+      ret["value"] = @value if @value
+      ret
+    end
+
+    def self.create_enabled(content_params)
```

Comment:
Look at  the 2 snippets below
https://github.com/Katello/katello/pull/6667/files#diff-beaa1cde8d4964406e1f688ae9de5befL164

https://github.com/Katello/katello/pull/6667/files#diff-65d2a2cbc038a622df645394a941f518L227

Both controllers have duplicated logic. I felt Model will be the right place to reuse this part and create the correct one.